draft-ietf-radext-radiusdtls-bis.md

```diff
@@ -541,6 +541,23 @@ These requirements reduce the possibility for a misbehaving client or server to
 
 This section discusses all specifications that are only relevant for RADIUS/TLS.
 
+## Sending and receiving RAIDUS traffic
+
+The TLS layer of RADIUS/TLS provides a stream-based communication between the two peers instead of the traditional packet-based communication as with RADIUS/UDP.
+As a result, the way RADIUS packets are sent and received has to change.
+
+Instead of relying on packet borders to indicate the start of a new packet, this information is carried implicitly in the stream.
```

Comment:
I don't think this is true?  Even RADIUS/UDP depended on the Length field.  RFC 2865 says:

```
      Octets outside the range of the Length field
      MUST be treated as padding and ignored on reception.
```

So the length of the packet is explicit, not implicit.

Perhaps just delete this sentence.

Reply:
I've reworded it to be more clear what I mean. I hope it's fixed with that.